docs/GettingStarted.md

```diff
@@ -32,6 +32,7 @@ This will build the following binaries:
                loading eBPF programs.
 * ebpfnetsh.dll: A plugin for the Windows netsh.exe command line tool that provides eBPF command line
                  utility functionality.
+* ebpfsvc.exe: A user-mode service which verifies and loads an eBPF program in the execution context.
```

Comment:
```suggestion
* ebpfsvc.exe: A user-mode service that verifies and loads an eBPF program in the execution context.
```
nit: synonymous in British english, but subtle difference in American english.  Chicago Manual of Style says "that" is correct here (in American English)